index.js

```diff
@@ -1,5 +1,6 @@
 'use strict'
 
+const { version } = require('./package.json')
```

Comment:
Cue the future issue where people transpiling everything will complain about needing the `package.json`....